edalize/apicula.py

```diff
@@ -0,0 +1,89 @@
+import os.path
+
+from edalize.edatool import Edatool
+from edalize.yosys import Yosys
+from importlib import import_module
+
+class Apicula(Edatool):
+
+    argtypes = ['vlogdefine', 'vlogparam']
+
+    @classmethod
+    def get_doc(cls, api_ver):
+        if api_ver == 0:
+            yosys_help = Yosys.get_doc(api_ver)
+            apicula_help = {
+                    'lists' : [
+                    	{'name' : 'device',
```

Comment:
device should be in `members` instead of `lists`. This way it will be a single value rather than a list of values. Compare e.g. with [the Quartus backend](https://github.com/olofk/edalize/blob/master/edalize/quartus.py#L30)